meta-balena-common/recipes-core/initrdscripts/files/finish

```diff
@@ -23,9 +23,16 @@ finish_run() {
 
         debug "Moving /dev, /proc and /sys onto rootfs..."
         mount --move /dev $ROOTFS_DIR/dev
-        mount --move /proc $ROOTFS_DIR/proc
         mount --move /sys $ROOTFS_DIR/sys
 
+	# remount proc in $ROOTFS_DIR instead of moving if systemd-bootchart is
+	# running, as it reads from /proc/schedstat
+	if pgrep systemd-bootchart; then
```

Comment:
Guessing `pgrep` is also included in the initramfs or it's an enabled busybox applet

Reply:
@alexgg `pgrep` is provided by busybox, is there a way to add this as an `RDEPENDS`?